### PR TITLE
feat: add optional reserveScrollBarGap

### PR DESCRIPTION
**Pull request by huksley**
improved #475 for more information see original PR
Optional reserveScrollBarGap={true}

### CODE REVIEW

**Comment by pradel** [comment]
@huksley thanks for the pr! Could you change the size so the CI pass please?

**Comment by huksley** [comment]
@pradel Done

File: react-responsive-modal/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-responsive-modal",
+  "name": "@huksley/react-responsive-modal",
```
Comment (pradel) [inline react-responsive-modal/package.json:2]: Can you revert this please?

**Review by pradel — CHANGES_REQUESTED** [review]

File: react-responsive-modal/package.json (hunk 1 above)
Comment (huksley) [reply react-responsive-modal/package.json]: omg, sorry I am using my package now. removed.

**Pull request merged**